modules/fargate/fargate/scaling.tf

```diff
@@ -0,0 +1,28 @@
+
+resource "aws_appautoscaling_target" "ecs_target" {
```

Comment:
This looks like it could be a standalone module I think. I.e. if we put the code under `modules/autoscaling` we could use it for regular ECS clusters, and also in `modules/fargate`?